app/plugins/mixinCommonMethods.js

```diff
@@ -114,6 +118,14 @@ export function showFrequency (frequency) {
   return showText
 }
 
+export function getCorrectSlug (item) {
```

Comment:
Note: it doesn't seem to be in use, which would make sense as we're introducing slugs to be returned by arcsi API.

Reply:
Yes, it looks like the frontend side handling of the previous item_slug idea.

Reply:
https://github.com/lahmacunradio/frontend/issues/254